src/app/beer_garden/log.py

```diff
@@ -9,10 +10,24 @@
 _LOGGING_CONFIG = None
 
 
-def setup_application_logging(config):
+def load(config: Box, force=False) -> dict:
```

Comment:
I'd say let's make this type hint a `dict` rather than a `Box`.

Reply:
:+1: